### DISCUSSION TITLE
MathJax support for Kanboard

### DISCUSSION THREAD

**Discussion by Zaunmich** [kanboard/kanboard#5187] [Q&A]
I found Issue #1348 [link](https://github.com/kanboard/kanboard/issues/1348) where it was talked about adding [MathJax](https://www.mathjax.org/) to the Kanboard via a Plugin.  But this discussion took place just before the Plugin system was implemented, and I was not able to find any Plugin.

The issue is quite old (last activity was in 2015) and now closed without the option to re-open it.

I went ahead and **created a Plugin which adds MathJax equation styling to Kanboard.**

You can check it out here: https://github.com/Zaunmich/mathjaxForKanboard

I am now curious how you get your Plugin added to the "official Plugin" list?

BR Michael

**Comment by jnahmias** [comment]
https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list
